### PR TITLE
Update insights logo

### PR DESCRIPTION
**Pull request by steverydz**
## Done

Update insights logo with blog logo

## QA

- Check out this feature branch
- Run the site using the command `./run serve`
- View the site locally in your web browser at: [http://0.0.0.0:8023/](http://0.0.0.0:8023/)
- Check that the logo does not include the word insights but does include the word blog

## Issue / Card

Fixes [#519](https://github.com/ubuntudesign/web-squad/issues/519)

### CODE REVIEW

File: static/sass/styles.scss [hunk 1]
```diff
@@ -238,3 +238,13 @@ pre {
 .u-margin-large--top {
   margin-top: $sp-large !important;
 }
+
+.p-navigation {
+
+  &__logo {
+    @media (min-width: $breakpoint-navigation-threshold + 1) {
+      margin-top: .75rem !important;
+      margin-bottom: .25rem !important;
```
Comment (anthonydillon) [inline static/sass/styles.scss]: We shouldn't need this. Can you file an issue on vanilla and add a comment here please.

Comment (steverydz) [reply static/sass/styles.scss]: Issue created: https://github.com/vanilla-framework/vanilla-framework/issues/1782

Comment (steverydz) [reply static/sass/styles.scss]: Sorry, I took that too literally :)
Done

**Comment by anasereijo** [comment]
@steverydz nice! thanks

**Comment by anthonydillon** [comment]
@steverydz looks like you have a lint issue

**Pull request merged**